front_end/panels/ai_chat/LLM/GenericOpenAIProvider.ts

```
@@ -0,0 +1,447 @@
+        return {
+          success: false,
+          message: 'Connection successful but no models found',
+        };
+      }
+
+      // If no model specified, use the first available model
+      const testModel = modelName || models[0].id;
+
+      // Try a simple completion to verify the API works
+      const testPrompt = 'Respond with "OK" to confirm connection.';
+      const response = await this.call(testModel, testPrompt, '', {
```

ESLint

[error] 'response' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)